### DISCUSSION TITLE
ViewSelector

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1001] [Q&A] [answered]
there have more revit file to bim file and have 3d viewer in every rvt file  
in ViewSelector, show the same view name from different rvt file!
how show the model or file name && view name
![image](https://user-images.githubusercontent.com/18022720/111866294-690dfc00-89a7-11eb-860d-872b77e24b48.png)

**Comment by pmconne** [comment, marked as answer]
I'm sorry but I do not understand your question. Please clarify.
